src/functions/template-tags/general.php

```diff
@@ -1097,8 +1098,8 @@ function tribe_events_event_schedule_details( $event = null, $before = '', $afte
 
 			if ( tribe_event_is_all_day( $event ) ) {
 				$inner .= tribe_get_start_date( $event, true, $format );
-				$inner .= '</span>' . $time_range_separator;
-				$inner .= '<span class="tribe-event-date-end">';
+				$inner .= ($html ? '</span>' : '') . $time_range_separator;
```

Comment:
Let's fix the spacing here
```suggestion
				$inner .= ( $html ? '</span>' : '' ) . $time_range_separator;
```